alm/scripts/createSnapshots.js

```diff
@@ -5,15 +5,41 @@ require('dotenv').config()
 const Web3 = require('web3')
 
 const fs = require('fs')
+const { parse } = require('dotenv')
 
 const {
   COMMON_HOME_RPC_URL,
   COMMON_HOME_BRIDGE_ADDRESS,
   COMMON_FOREIGN_RPC_URL,
-  COMMON_FOREIGN_BRIDGE_ADDRESS
+  COMMON_FOREIGN_BRIDGE_ADDRESS,
+  FOREIGN_BRIDGE_RPC_BLOCK_LIMIT,
+  HOME_BRIDGE_RPC_BLOCK_LIMIT,
+  FOREIGN_BRIDGE_DEPLOYED_AT_BLOCK
 } = process.env
 
-const generateSnapshot = async (side, url, bridgeAddress) => {
+const getPastEventsLimited = async (contract, eventName, fromBlock, toBlock, blockLimit) => {
+  let events = []
+  const chunks = parseInt(blockLimit)
+
+  if (chunks === -1) {
```

Comment:
it can be optional param, with undefined.

Reply:
why also `blockLimit` is renamed to chunks after parseInt?
I think something like `blockSize` or `batchSize` suites more

Reply:
https://github.com/fuseio/tokenbridge/pull/6/commits/2f94988c3ccb35c167396518d714facb4551e936
is this better?